### ISSUE TITLE
[Feature]: IP whitelist/blacklisting incoming connection

### ISSUE THREAD

**Issue by AriaFantom** [coollabsio/coolify#832]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

The feature would be like :-

1) Click on specific service
2) While allowing Public Connection and there to mention an IP range or Specific IP
3) If they set to 0.0.0.0 it will allow all connections and if they set an Ip range or specific IP it will allow only allow those ip to connect 

### Why should this be worked on?

I have seen this type of feature in MongoDB Atlas and i think it would be a great feature in this project which also can give the container extra security.

**Comment by BassoliCodes** [comment]
I think it's a great idea. It could even have a part in the panel where we can see the connections that are being made, IP addresses that are making attacks among others, it would be a way of protection both for Coolify and for our applications and services hosted on it.

Of course, the whole protection part, whoever did the installation had to configure

**Comment by AriaFantom** [comment]
> 

Yes true because mainly the issue is the MongoDB has some vulnerability which some botnets use to bruteforce mongo running services and they just shut it down or just corrupt all the data

And also if we make an Priavte API which i want to be access only by specific ips then this feature will come in very handy :) rather than implementing our own logic

**Comment by andrasbacsai** [comment]
v4 will have iptables support.

I'm closing this issue, because tracking features will be moved to GitHub discussions. Github issues will be only for bugs.

Version 4 is in full focus. v3 related issues will be closed, if it is not a critical bug.

Thanks for your understanding.
